finetune/base_models/bert/model.py

```diff
@@ -323,3 +330,29 @@ class DistilRoBERTa(_BaseBert):
             "url": urljoin(GPT2_BASE_URL, "encoder.json"),
         }
     ]
+
+
+class LayoutLM(_BaseBert):
+    encoder = LayoutLMEncoder
+    featurizer = layoutlm_featurizer
+    settings = dict(DocRep.settings)
```

Comment:
Honestly I think it's preferable to redefine this manually.  I wouldn't expect that modifying DocRep settings would have downstream effects and I can see this causing issues later as we work more on DocRep.